notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/DistCopyJob.scala

```diff
@@ -89,11 +91,14 @@ class DistCopyMapper extends Mapper[NullWritable, Mapping, NullWritable, NullWri
   var readLimit: Int = 0
   var multipartUploadThreshold: Long = 0
   var retryCount: Int = 0
+  var crossValidate: Boolean = true
 
   override def setup(context: Mapper[NullWritable, Mapping, NullWritable, NullWritable]#Context): Unit = {
     // get the default mapper parameter values
     val parameters = DistCopyMapperParameters.Default
 
+    crossValidate =
+      context.getConfiguration.getBoolean(CrossValidate, false)
```

Comment:
Should this be defaulting to `false`? i know it should never fallback to default, but in the case where it doesn't get set correctly, is it safer for it to be set to `true`?